### PR TITLE
fix versioning gh aciton

### PR DESCRIPTION
**Pull request by bangtoven**
### _How did you test your changes?_

https://github.com/coinbase/coinbase-wallet-sdk/pull/999

### CODE REVIEW

File: .github/workflows/versioning.yml [hunk 1]
```diff
@@ -41,8 +41,11 @@ jobs:
           cache-dependency-path: '**/yarn.lock'
 
       - name: Update version
-        run: yarn version --new-version ${{ env.PACKAGE_VERSION }} --no-git-tag-version
+        run: npm version ${{ env.PACKAGE_VERSION }} --no-git-tag-version --workspaces-update=false
```
Comment (bangtoven) [inline .github/workflows/versioning.yml:44]: `yarn version` is supported only on yarn 1.xx
as we bumped yarn version to 3.x, it no longer works.
tried multiple combinations of changes, but this seems to be the simplest solution.
for example, i tried `yarn plugin import version`, but it didn't allow to set the whole version value which this gh action is about. (we might migrate to this new method later)

File: .github/workflows/versioning.yml [hunk 2]
```diff
@@ -41,8 +41,11 @@ jobs:
...
 
+      - name: Install NPM dependencies
+        run: yarn install --immutable
```
Comment (bangtoven) [inline .github/workflows/versioning.yml:47]: this is required to run any yarn command. applying the same changes made to main.yml gh action: https://github.com/coinbase/coinbase-wallet-sdk/commit/c7442551178e4a19254258796a8c791f6d6f8d11

File: packages/wallet-sdk/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@coinbase/wallet-sdk",
-  "version": "3.7.1",
+  "version": "3.9.0-canary.0",
```
Comment (bangtoven) [inline packages/wallet-sdk/package.json:3]: verified: https://github.com/coinbase/coinbase-wallet-sdk/pull/999

**Review by cb-jake — APPROVED** [review]

**Review by jonesk7734 — COMMENTED** [review]
Ok

**Pull request merged**